essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-use-case-software-engineer.md

```diff
@@ -1,2 +1,87 @@
 # \[Draft] Use Case: Software Engineer
 
+
+<!-- Introduction -->
+## Introduction
+Welcome, fellow software engineers! Today, I want to share with you how I use Appflowy to stay organized and productive in my work. As software engineers, we all know how important it is to keep track of our projects, deadlines, and tasks. That's where Appflowy comes in! With its simple and intuitive design, Appflowy makes it easy to stay on top of everything.
+
+![image](https://user-images.githubusercontent.com/89600478/235852123-0105d13e-b05e-4df0-b136-4825ca817351.png)
+
+
+
+<!-- How to Contribute -->
+## Installation
+These are the installation instructions for an end user. If you wish to contribute to AppFlowy or if you just want to view and run directly from source code please visit the [Contribute to AppFlowy section](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy).<br>
```

Comment:
I would like to revise it in the following way.
//
Before the installation, please make sure your machine meets the [System Requirements](https://appflowy.gitbook.io/docs/essential-documentation/install-appflowy/requirements). Here are the [Installation methods](https://appflowy.gitbook.io/docs/essential-documentation/install-appflowy/installation-methods) that will help you install Appflowy.
If you would like to contribute to AppFlowy or access its source code, please visit the [Contribute to AppFlowy section](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy).
//